app/controllers/admin/administrators/administrators_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Admin
+  module Administrators
+    class AdministratorsController < Admin::BaseController
+      skip_after_action :verify_authorized, only: :index
+      skip_after_action :verify_policy_scoped, except: :index
+
+      def index
+        @administrators = policy_scope(User).admins
+      end
+
+      def new
+        authorize AdminProfile
+
+        @user = User.new(full_name: params[:full_name], email: params[:email])
```

Comment:
Can we have a method for permitted params? Also, why are params here not coming though `require(:user)`? Are they url params?